sdk-setup/src/mb2

```diff
@@ -3071,7 +3072,10 @@ run_deploy() {
             fatal "Nothing to deploy. Maybe you forgot to use the \"make-install\" command."
         fi
 
-        if [[ -e $INSTALL_STAMP && $buildroot -nt $INSTALL_STAMP ]]; then
+        if [[ -e $TMP_INSTALL_STAMP ]]; then
+            [[ $buildroot -nt $TMP_INSTALL_STAMP ]] && \
+                fatal "The content of the RPM \"%buildroot\" directory does not match this build."
+        elif [[ -e $INSTALL_STAMP && $buildroot -nt $INSTALL_STAMP ]]; then
```

Comment:
We could better store the timestamp as file data in run_make_install

```
stat -c %Y "$buildroot" > "$INSTALL_STAMP"
```

and then

```suggestion
        if [[ -e $INSTALL_STAMP && $(stat -c %Y "$buildroot") -ne $(<"$INSTALL_STAMP") ]]; then
```

Reply:
Right. I'll test with that.

Reply:
Seems to work